component/Page/Profile/index.tsx

```diff
@@ -1,5 +1,6 @@
 import Image from "next/image";
 import LayoutContainer from "@components/LayoutContainer";
+import LinkableText from "@components/LinkableText/LinkableText";
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of the new `LinkableText` component is correctly added. No issues found.
```diff
+ import LinkableText from "@components/LinkableText/LinkableText";
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->